.github/workflows/test.yml

```
@@ -27,123 +27,60 @@ jobs:
+      - name: Upload Coverage Results
+        uses: codecov/codecov-action@v6
         with:
-          name: python-bindings # TBM: use a different namespace than in build-packages for now.
-          path: packages/python/
-      - id: python-versions
-        name: Export Python Versions to Environment
-        run: |
-          python_versions=$(echo -n "["; find packages/python -name "*.whl" | grep -o "py[0-9]\{2,3\}" | sed 's/py\([0-9]\)\([0-9][0-9]\?\)/\1.\2/' | sed "s/\([0-9.]*\)/'\1'/" | tr '\n' ',' | sed 's/,$//'; echo "]")
-          echo "Python versions: ${python_versions}"
-          echo "value=${python_versions}" >> $GITHUB_OUTPUT
+          token: ${{ secrets.CODECOV_TOKEN }}
```

actionlint (1.7.12)

[error] property "codecov_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; dockerhub_token: string; dockerhub_username: string; github_token: string}

(expression)